Solutions/Recorded Future/Package/createUiDefinition.json

```diff
@@ -78,13 +78,16 @@
             "elements": [
               {
                 "name": "workbook1-text",
-                "type": "Microsoft.Common.TextBlock"
+                "type": "Microsoft.Common.TextBlock",

```

Comment:
ARM-ttk is failing because of these white spaces. Can you please remove this?
Thanks